bounties/npm/node-influx/1/README.md

```diff
@@ -0,0 +1,13 @@
+# Overview
+
+`influx` is an InfluxDB Client for Node.js and Browsers.
```

Comment:
A fix has been implemented in https://github.com/node-influx/node-influx/commit/dcc2e7e9564e0303a771a1402ac59a46d6a0a082